src/modules/GroupChannel/components/MessageList/getMessagePartsInfo.ts

```diff
@@ -39,9 +39,9 @@ export const getMessagePartsInfo = ({
   const previousMessageCreatedAt = previousMessage?.createdAt;
   const currentCreatedAt = currentMessage.createdAt;
   // https://stackoverflow.com/a/41855608
-  const hasSeparator = !(previousMessageCreatedAt && (
-    isSameDay(currentCreatedAt, previousMessageCreatedAt)
-  ));
+  const hasSeparator = 'sendingStatus' in currentMessage && currentMessage.sendingStatus !== 'succeeded' // NOTE: for pending/failed messages
+    ? false
+    : !(previousMessageCreatedAt && (isSameDay(currentCreatedAt, previousMessageCreatedAt)));
```

Comment:
Can we wrap on the ternary operator with parentheses? for readability
```suggestion
  const hasSeparator = 'sendingStatus' in currentMessage && (
    currentMessage.sendingStatus !== 'succeeded'
      ? false
      : !(previousMessageCreatedAt && (isSameDay(currentCreatedAt, previousMessageCreatedAt)))
      );
```

Reply:
sure~